### DISCUSSION TITLE
Do you need to flash the screen with a DWIN_SET?

### DISCUSSION THREAD

**Discussion by jjfs127** [Jyers/Marlin#32] [Q&A] [answered]
I'm already running Smith3d firmware and would like to give your firmware a whirl. Do I have to compile a new DWIN_SET for the LCD (if so, how do I accomplish this?)
Thanks

**Comment by chevyman142000** [comment]
I don't have the exact answer for you, but wanted to let you know that I was using TH3D firmware prior to this and did not see any issues when I went to this firmware.

**Comment by Jyers** [comment, marked as answer]
No, you do not have to compile a DWIN_SET for this firmware.

**Reply by isochromanone** [reply]
Is DWIN something that one only updates if there's an issue?  I've never been clear on when to do it.

**Reply by Jyers** [reply]
Updating the screen with a DWIN_SET is only if you want to change the icons
